foundations/foundation-config/src/main/java/org/apache/servicecomb/config/ConfigUtil.java

```diff
@@ -52,9 +52,15 @@
 
   private static final String MICROSERVICE_CONFIG_LOADER_KEY = "cse-microservice-config-loader";
 
+  private static ConfigModel model = new ConfigModel();
+
   private ConfigUtil() {
   }
 
+  public static void setConfigs(Map<String, Object> config) {
```

Comment:
I think if we can provide a more convenient method to add configs . e.g. 
public static void addConfig(String k, Object v)

Reply:
you mean we provide both addConfig and setConfigs? or just replace setConfigs ?